packages/components/src/animate/index.js

```diff
@@ -1,39 +1,69 @@
 /**
  * External dependencies
  */
-import classnames from 'classnames';
+import { isFunction } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import deprecated from '@wordpress/deprecated';
+
+/**
+ * Internal dependencies
+ */
+import {
+	AppearingWrapper,
+	SlidingWrapper,
+	LoadingWrapper,
+} from './styles/animate-styles';
 
 function Animate( { type, options = {}, children } ) {
+	/**
+	 * For backwards compatability we need to handle when
+	 * children as a function is passed in. By passing an
+	 * empty string for className we should prevent any
+	 * conventional usage of this component from before the
+	 * change from breaking.
+	 */
+	const safeChildren = isFunction( children )
+		? () => children( { className: '' } )
+		: () => children;
+
 	if ( type === 'appear' ) {
 		const { origin = 'top' } = options;
 		const [ yAxis, xAxis = 'center' ] = origin.split( ' ' );
 
-		return children( {
-			className: classnames( 'components-animate__appear', {
-				[ 'is-from-' + xAxis ]: xAxis !== 'center',
-				[ 'is-from-' + yAxis ]: yAxis !== 'middle',
-			} ),
-		} );
+		if ( yAxis === 'middle' ) {
```

Comment:
I believe this value was used for consistency with the Popover position props but I may be wrong.

Reply:
Hmm, should it be kept and still filtered out then? It does appear to line up with the Popover position props.